swift-extensions/UITextViewExtensions.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+import TRIKOT_FRAMEWORK_NAME
+
+extension UITextView {
+    public var labelViewModel: LabelViewModel? {
+        get { trikotViewModel() }
+        set(value) {
+            viewModel = value
+            if let labelViewModel = value {
+                bind(labelViewModel.text, \UITextView.text)
```

Comment:
Do you think we should support richText, just like we do for [UILabel](https://github.com/mirego/trikot.viewmodels/blob/master/swift-extensions/UILabelExtensions.swift)?

Reply:
You are right, we should. I did not initially because in my use case I did not need it. I will add it now